themes/digital.gov/src/js/common/edit-this.js

```diff
@@ -22,12 +22,20 @@ jQuery(document).ready(function($) {
 			// remove the active class from the edit button
 			$('.edit_tools').removeClass('active');
 			// swap out the icon in the edit button
-			$(this).html('<i class="far fa-edit"></i>');
+			$(this).html('<i class="far fa-edit"></i><span class="screenreaderonly">Toggle edit this page</span>');
 		} else {
 			enable_edit_this();
 			$('.edit_tools').addClass('active');
-			$(this).html('<i class="fas fa-times"></i>');
+			$(this).html('<i class="fas fa-times"></i><span class="screenreaderonly">Toggle edit this page</span>');
 		}
   });
 
 });
+
+$(function(){
+    $('.edit-issue').keypress(function(e) {
+        if(e.keyCode == 32){
+			window.open("https://github.com/GSA/digitalgov.gov/issues/new?body=%0D%0A%2A%2A{{- .Title -}}%2A%2A%0D%0Ahttps%3A%2F%2Fdigital.gov%2F{{- .Permalink -}}%0D%0A%0D%0A---%0D%0A%0D%0A%2A%2APlease+describe+the+issue+clearly.+Include+a+screenshot+if+needed%2A%2A%0D%0A")
```

Comment:
Indents seem wrong and show as tabs instead of spaces.

Alternatively, you could trigger a click via `$el.trigger("click")` instead of `window.open`.

Reply:
I believe this how the file was when I got it. I changed tabs to two spaces.